### ISSUE TITLE
WebView flavor issue

### ISSUE THREAD

**Issue by sudophils** [eliataylor/tam-flutter#1]
I have reviewed your code as posted in the freelance upwork,  i can fix  this issue for you, from my looks, at the repo, there seems to be obvious issues unless the code is not complete as shared here, are you also targeting to run it on ios (or it already work on iOS?), since the error you shared is android specific

**Comment by eliataylor** [comment]
yes @sudophils , i can build any app, but simply replacing the values in .env and recompiling.

the still getting an issue with Android / Gradle.

**Comment by sudophils** [comment]
I can fix this for you and you can move forward.  I would have bid on the project but I have pending proposals, find with with Philips N Flutter developer.

**Comment by sudophils** [comment]
<img width="251" alt="Screen Shot 2023-04-04 at 5 10 18 AM" src="https://user-images.githubusercontent.com/5889921/229744679-5cb2664f-5d52-4849-8a59-251bf74638c7.png">
I have gone pass that initial build error, this error is a different one but cheaper i bet

**Comment by eliataylor** [comment]
what's your email @sudophils ?

**Comment by sudophils** [comment]
> what's your email @sudophils ?

philips.nge@gmail.com

**Comment by eliataylor** [comment]
The latest commit can be compiled into any App brand by 1. replacing .env with the .env.[brand] and 2. only for Android, it's necessary to hardcode the respective `package` name in android/app/src/main/AndroidManifest.xml

To avoid the hardcoded change for Android, maybe this approach might work for both platforms:
- https://github.com/ByneappLLC/flutter_config/issues/35 (another tutorial on `dart-define` https://thiele.dev/blog/flutter-dart-define-part-2-dev-and-prod-package-names-and-bundle-ids/)

However, for now, i think this is a suitable solution for keeping everything in the same codebase for now.
